fta/samples/api/views.py

```diff
@@ -44,3 +45,56 @@ def add_sample(self, request, format=None):
         sample = sample_from_required(frozen_page, freeze_software, notes)
         sample.save()
         return Response({"id": sample.id})
+
+
+class AddFathomSampleViewSet(viewsets.ViewSet):
+    basename = "add_fathom_sample"
+
+    @action(detail=False, methods=["post"])
+    def add_fathom_sample(self, request, format=None):
```

Comment:
I think `add_labeled_sample` might be a better name for this API. All of the FTA samples are in a way "Fathom samples", but `add_labeled_sample` makes it clear what's the distinction between this API endpoint and `add_sample` above: whether or not the sample has labels.

Reply:
Agreed